db/dictionary_es.csv

```diff
@@ -1,398 +1,398 @@
-key,text,english,notes
-version,4.5,,
-pdf_title,6000 años de historia humana visualizados,6000 years human history visualized,
-pdf_subject,Cronología de la humanidad,Timeline of humankind,
-BCE,a.e.c. ,B.C.E.,
-CE,e.c. ,C.E.,
-year,año,year,"timespan, age (Adam was 130 years old when Shem was born)"
-years,años,years,
-years_age,años,years,years when used as age for a person (different word than years in some languages)
-age_kings,años, ,"if the word ""year"" appears after the numeral, like ""at the age of 35 years"""
-month,mes,month,
-months,meses,months,
-day,día,day,
-days,días,days,
-to,a,to,
-people,personas,persons,
-judges,jueces,judges,
-prophets,profetas,prophets,
-kings,reyes,kings,
-periods,periodos,periods,
-events,eventos,events,
-objects,cosas o artículos,things or items,
-Deluge,Diluvio,Deluge,
-Adam,Adán,Adam,
-Seth,set,Seth,
-Enosh,Enós,Enosh,
-Kenan,Kenán,Kenan,
-Mahalalel,ma·hala·lel,Ma·hala·lel,
-Jared,Jared,Jared,
-Enoch,Enoc,Enoch,
-Methuselah,Matusalén,Me·thuse·lah,
-Lamech,Lamec,Lamech,
-Noah,Noé,Noah,
-Shem,Sem,Shem,
-Arpachshad,Ar·pajshad,Ar·pachshad,
-Shelah,Sela,Shelah,
-Eber,Éber,Eber,
-Peleg,Péleg,Peleg,
-Reu,Reu,Reu,
-Serug,Serug,Serug,
-Nahor,Nacor,Nahor,
-Terah,Taré,Terah,
-Abraham,Abrahán,Abraham,
-Isaac,isaac,Isaac,
-Jacob,jacob,Jacob,
-Joseph,Joseph,Joseph,
-Moses,Moisés,Moses,
-Othniel,Otoniel,Othniel,
-Ehud,Aod,Ehud,
-Shamgar,Samgar,Shamgar,
-Barak,Barak,Barak,
-Gideon,Gedeón,Gideon,
-Tola,tolá,Tola,
-Jair,jair,Jair,
-Jephthah,Jefté,Jephthah,
-Ibzan,Ibzán,Ibzan,
-Elon,elon,Elon,
-Abdon,Abdón,Abdon,
-Samson,Sansón,Samson,
-Samuel,Samuel,Samuel,
-Saul,Saúl,Saul,
-David,David,David,
-Solomon,Salomón,Solomon,
-Rehoboam,Roboam,Rehoboam,
-Abijah,Abías (Abiam),Abijah (Abijam),
-Asa,Como un,Asa,
-Jehoshaphat,Josafat,Jehoshaphat,
-Jehoram2,Joram,Jehoram,
-Ahaziah,Ocozías,Ahaziah,
-Athaliah,Reina Atalía,Queen Athaliah,
-Jehoash,Joás,Jehoash,
-Amaziah,Amasías,Amaziah,
-Uzziah,Uzías (Azarías),Uzziah (Azariah),
-Jotham,Jotam,Jotham,
-Ahaz,Acaz,Ahaz,
-Hezekiah,Ezequías,Hezekiah,
-Manasseh,Manasés,Manasseh,
-Amon,Amón,Amon,
-Josiah,Josías,Josiah,
-Jehoahaz,Joacaz,Jehoahaz,
-Jehoiakim,Joacim,Jehoiakim,
-Jehoiachin,Joaquín,Jehoiachin,
-Zedekiah,Sedekia,Zedekiah,
-Jeroboam,Botella grande,Jeroboam,
-Nadab,Nadab,Nadab,
-Baasha,Baasa,Baasha,
-Elah,Ela,Elah,
-Zimri,Zimrí,Zimri,
-OmriTibni,Omri y Tibni juntos,Omri and Tibni together,
-Omri,Omrí (solo),Omri (alone),
-Ahab,Acab,Ahab,
-Ahaziah,Ocozías,Ahaziah,
```

Comment:
Ahaziah seems to be twice in the dictionary

Reply:
Along with Korah, Nahor and Zephaniah